src/config/cambodia/layers.json

```diff
@@ -1656,10 +1661,6 @@
       "Commune": {
         "type": "text",
         "label": "Commune"
-      },
-      "NumPeoAff": {
```

Comment:
I noticed that removing this breaks the "measure" field field of the popup data, not sure why